src/www/messages/fa.json

```diff
@@ -71,9 +72,7 @@
   "server-added": "سرور «{serverName}» اضافه شد!",
   "server-connected": "به «{serverName}» وصل شدید",
   "server-create-your-own": "کلید دسترسی ندارید؟<br>کلید دسترسی خودتان را در {openLink}وبسایت ما{closeLink} ایجاد کنید.",
-  "server-create-your-own-access": "کلید دسترسی ندارید؟<br>کلید دسترسی خودتان را در {openLink}وبسایت ما{closeLink} ایجاد کنید یا <br>یک {openLink2}کلید دسترسی{closeLink} درخواست کنید.",
```

Comment:
Is this deletion intended?

Reply:
Are we using `server-create-your-own-access` and `server-create-your-own-zero-state-access`?